docs/intro/from-ground-up.rst

```diff
@@ -301,22 +292,35 @@ It might be easier to read the code written this way. Also, this style
 allows to extract only some of the attributes - if you don't need
 the complete to_item() output, you still can access individual properties.
 
-.. note::
-    web-poet provides a small framework to simplify writing Page Objects
-    in this style; see :ref:`web-poet-fields` .
-
-You may even write some base class to make it nicer - e.g. helper descriptors
-to define properties from CSS selectors, and a default ``to_item``
-implementation (so, no need to define ``to_item``).
-This is currently not implemented in ``web-poet``, but
-nothing prevents us from having a DSL like this:
+web-poet provides a small framework to simplify writing Page Objects
+in this style; see :ref:`web-poet-fields`. The example above can be simplified
+using web-poet fields - there is no need to write ``to_item`` boilerplate:
```

Comment:
```suggestion
using web-poet :func:`web_poet.fields` - there is no need to write ``to_item`` boilerplate:
```

Reply:
web-poet fields are referred here as a concept, not as a module; I'm not sure it adding a link helps. The link doesn't look right (web_poet.fields is a module, not a function).